{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at AuthenticationService.authenticate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:117:8)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at Route.dispatch (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:277:22\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)\n    at Function.handle (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:176:3)\n    at router (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:46:12)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:312:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:280:7\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T09:38:44.910Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at AuthenticationService.authenticate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:117:8)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at Route.dispatch (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:277:22\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)\n    at Function.handle (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:176:3)\n    at router (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:46:12)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:312:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:280:7\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T09:38:44.920Z"}
{"error":"ValidationError: Activity validation failed: due_to: Cast to Date failed for value \"Invalid Date\" at path \"due_to\"\n    at new ValidationError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\validation.js:27:11)\n    at model.Document.invalidate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:1758:32)\n    at model.$set (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:817:10)\n    at model._handleIndex (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:639:14)\n    at model.$set (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:600:22)\n    at model.Document (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:105:12)\n    at model.Model (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:61:12)\n    at new model (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3771:13)\n    at toExecute.push.callback (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:2059:22)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\internal\\parallel.js:27:9\n    at eachOfArrayLike (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\eachOf.js:57:9)\n    at exports.default (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\eachOf.js:9:5)\n    at _parallel (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\internal\\parallel.js:26:5)\n    at parallelLimit (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\parallel.js:85:26)\n    at utils.promiseOrCallback.cb (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:2069:5)\n    at Object.promiseOrCallback (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\utils.js:211:14)","level":"error","message":"FatalError","timestamp":"2018-05-22T09:39:54.952Z"}
{"error":"TypeError: Cannot read property '_id' of undefined\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\routes\\activities.js:45:23\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:181:32\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T09:39:54.972Z"}
{"error":"ValidationError: Activity validation failed: due_to: Cast to Date failed for value \"NaN\" at path \"due_to\"\n    at new ValidationError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\validation.js:27:11)\n    at model.Document.invalidate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:1758:32)\n    at model.$set (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:817:10)\n    at model._handleIndex (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:639:14)\n    at model.$set (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:600:22)\n    at model.Document (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\document.js:105:12)\n    at model.Model (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:61:12)\n    at new model (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3771:13)\n    at toExecute.push.callback (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:2059:22)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\internal\\parallel.js:27:9\n    at eachOfArrayLike (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\eachOf.js:57:9)\n    at exports.default (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\eachOf.js:9:5)\n    at _parallel (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\internal\\parallel.js:26:5)\n    at parallelLimit (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\node_modules\\async\\parallel.js:85:26)\n    at utils.promiseOrCallback.cb (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:2069:5)\n    at Object.promiseOrCallback (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\utils.js:211:14)","level":"error","message":"FatalError","timestamp":"2018-05-22T09:40:39.743Z"}
{"error":"TypeError: Cannot read property '_id' of undefined\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\routes\\activities.js:45:23\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:181:32\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T09:40:39.764Z"}
{"error":"Error: Argument passed in must be a single String of 12 bytes or a string of 24 hex characters\n    at new ObjectID (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\bson\\lib\\bson\\objectid.js:51:11)\n    at ObjectID [as ObjectId] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\bson\\lib\\bson\\objectid.js:32:43)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\routes\\activities.js:31:38\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:181:32\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T09:41:34.352Z"}
{"error":"TypeError: Cannot read property '_id' of undefined\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\routes\\activities.js:45:23\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:181:32\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T09:41:34.390Z"}
{"error":"TypeError: Cannot read property '_id' of undefined\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\routes\\activities.js:45:23\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:181:32\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T09:42:51.917Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:247:13\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T09:54:39.460Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:156:13\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T09:54:39.471Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:156:13\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T09:54:39.479Z"}
{"error":"TypeError: Cannot read property '_id' of undefined\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\routes\\activities.js:45:28\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:181:32\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T09:57:29.195Z"}
{"error":"TypeError: Cannot read property '_id' of undefined\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\routes\\activities.js:45:28\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:181:32\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:01:14.651Z"}
{"error":"TypeError: Cannot read property '_id' of undefined\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\routes\\activities.js:45:28\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:181:32\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\model.js:3949:16\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:2016:28)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:02:08.126Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at AuthenticationService.authenticate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:125:9)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at Route.dispatch (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:277:22\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)\n    at Function.handle (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:176:3)\n    at router (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:46:12)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:312:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:280:7\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T10:12:48.198Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at AuthenticationService.authenticate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:125:9)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at Route.dispatch (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:277:22\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)\n    at Function.handle (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:176:3)\n    at router (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:46:12)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:312:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:280:7\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T10:12:48.229Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at AuthenticationService.authenticate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:125:9)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at Route.dispatch (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:277:22\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)\n    at Function.handle (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:176:3)\n    at router (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:46:12)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:312:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:280:7\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T10:12:48.238Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at AuthenticationService.authenticate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:125:9)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at Route.dispatch (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:277:22\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)\n    at Function.handle (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:176:3)\n    at router (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:46:12)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:312:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:280:7\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T10:28:13.356Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at AuthenticationService.authenticate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:125:9)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at Route.dispatch (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:277:22\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)\n    at Function.handle (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:176:3)\n    at router (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:46:12)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:312:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:280:7\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T10:28:13.399Z"}
{"error":"    at new CustomError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\misc\\Error.js:7:14)\n    at AuthenticationService.authenticate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:125:9)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at Route.dispatch (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:277:22\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)\n    at Function.handle (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:176:3)\n    at router (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:46:12)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:312:13)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:280:7\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)","level":"error","message":"InvalidRequest","timestamp":"2018-05-22T10:28:13.412Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:32:02.885Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:32:02.892Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:33:29.596Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:33:29.603Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:35:19.519Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:35:19.527Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:35:45.907Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:35:45.914Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:36:24.065Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:36:24.072Z"}
{"error":"CastError: Cast to date failed for value \"lol\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:300:32)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:37:33.055Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:37:33.063Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:39:43.327Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:39:43.335Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:40:13.107Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:40:13.115Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:41:51.953Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:41:51.961Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:41:53.782Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:41:53.786Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:49:07.664Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:49:07.672Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:50:44.453Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:50:44.461Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:51:46.630Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:51:46.636Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:52:46.291Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:52:46.297Z"}
{"error":"CastError: Cast to date failed for value \"{ wtf: 'yes' }\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:53:10.224Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:53:10.231Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:58:31.417Z"}
{"error":"CastError: Cast to date failed for value \"{}\" at path \"due_to\" for model \"Activity\"\n    at new CastError (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\error\\cast.js:27:11)\n    at SchemaDate.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:248:9)\n    at SchemaDate.SchemaType.applySetters (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:724:12)\n    at SchemaDate.SchemaType._castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1095:15)\n    at SchemaDate.castForQuery (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schema\\date.js:282:17)\n    at SchemaDate.SchemaType.castForQueryWrapper (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\schematype.js:1064:15)\n    at cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\cast.js:245:34)\n    at model.Query.Query.cast (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:3210:12)\n    at model.Query.Query._castConditions (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1280:10)\n    at model.Query.Query._find (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\mongoose\\lib\\query.js:1295:8)\n    at process.nextTick (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\kareem\\index.js:311:33)\n    at _combinedTickCallback (internal/process/next_tick.js:131:7)\n    at process._tickCallback (internal/process/next_tick.js:180:9)","level":"error","message":"FatalError","timestamp":"2018-05-22T10:58:31.424Z"}
{"error":"SyntaxError: Unexpected token o in JSON at position 1\n    at JSON.parse (<anonymous>)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\routes\\activities.js:12:22\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at AuthenticationService.optionalAuthenticate (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\app\\services\\AuthenticationService.js:208:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:131:13)\n    at Route.dispatch (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:277:22\n    at Function.process_params (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:330:12)\n    at next (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:271:10)\n    at Function.handle (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:176:3)\n    at router (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:46:12)\n    at Layer.handle [as handle_request] (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\sfast\\Documents\\IMT18\\project\\server\\node_modules\\express\\lib\\router\\index.js:312:13)","level":"error","message":"FatalError","timestamp":"2018-05-22T11:00:19.696Z"}
